Sources/BetaToolkit/AppDeployToolkitMain.ps1

```diff
@@ -3516,7 +3516,8 @@ Function Get-UserProfiles {
 			ForEach-Object {
 				Get-ItemProperty -LiteralPath $_.PSPath -ErrorAction 'Stop' | Where-Object { ($_.ProfileImagePath) } |
 				Select-Object @{ Label = 'NTAccount'; Expression = { $(ConvertTo-NTAccountOrSID -SID $_.PSChildName).Value } }, @{ Label = 'SID'; Expression = { $_.PSChildName } }, @{ Label = 'ProfilePath'; Expression = { $_.ProfileImagePath } }
-			}
+			} |
+			Where-Object { $_.NTAccount -notmatch 'defaultuser0$'} # This excludes the "defaultuser0" account used by CloudExperienceHostBroker.exe
```

Comment:
I'd suggest matching against `defaultuser\d+$`, as its index isn't always 0.

Reply:
True, but defaultuser0 is the one used during OOBE.
@seanlillis Do you agree with my suggestion?